sky/skylet/job_lib.py

```diff
@@ -229,49 +228,45 @@ def query_job_status(job_owner: str, job_ids: List[int]) -> List[JobStatus]:
     Though we update job status actively in ray program and job cancelling,
     we still need this to handle staleness problem, caused by instance
     restarting and other corner cases (if any).
+
+    This function should only be run on the remote instance with ray==1.13.0.
     """
     if len(job_ids) == 0:
         return []
 
     # TODO: if too slow, directly query against redis.
     ray_job_ids = [make_ray_job_id(job_id, job_owner) for job_id in job_ids]
-    test_cmd = [
-        (
-            f'(ray job status --address 127.0.0.1:8265 {ray_job_id} 2>&1 | '
-            # Not a typo, ray has inconsistent output for job status.
-            # succeeded: Job 'job_id' succeeded
-            # running: job 'job_id': RUNNING
-            # stopped: Job 'job_id' was stopped
-            # failed: Job 'job_id' failed
-            f'grep "ob \'{ray_job_id}\'" || echo "not found")')
-        for ray_job_id in ray_job_ids
-    ]
-    test_cmd = ' && '.join(test_cmd)
-    proc = subprocess.run(test_cmd,
-                          shell=True,
-                          check=True,
-                          executable='/bin/bash',
-                          stdout=subprocess.PIPE)
-    stdout = proc.stdout.decode('utf-8')
 
-    results = stdout.strip().split('\n')
-    assert len(results) == len(job_ids), (results, job_ids)
+    from ray import job_submission  # pylint: disable=import-outside-toplevel
```

Comment:
try catch ImportError and print a message including the current `ray.__version__`?